src/mattack_actors.cpp

```diff
@@ -170,6 +170,13 @@ void melee_actor::load_internal( JsonObject &obj, const std::string & )
               _( "The %1$s bites <npcname>'s %2$s, but fails to penetrate armor!" ) );
     optional( obj, was_loaded, "hit_dmg_npc", hit_dmg_npc, translated_string_reader,
               _( "The %1$s bites <npcname>'s %2$s!" ) );
+    optional( obj, was_loaded, "miss_msg_npc", miss_msg_monster, translated_string_reader,
+              _( "The %s lunges a the <npcname>, but they dodge!" ) );
```

Comment:
Should be "at," and since this is an NPC's name you can drop "the"

(The bear lunges at the Trevor, but they dodge!)

Reply:
This message was supposed to only trigger for non humanoid monsters, I kept in the <npcname> part because that also resolves to the monster name. (Not sure if there is a way to replace this with <monstername>). (I missed renaming the 'miss_msg_npc' to 'miss_msg_monster' btw).